modules/gdscript/gdscript_tokenizer.cpp

```diff
@@ -340,7 +340,7 @@ StringName GDScriptTokenizer::get_token_literal(int p_offset) const {
 				default: {
 				}
 			}
-		}
+		} break;
```

Comment:
@vnen This changes the logic, but I think it's fixing a bug?

Clang's warning was a bit weird as it pointed to the next line, so for a few minutes I thought it didn't like two consecutive labels :/
```
modules/gdscript/gdscript_tokenizer.cpp:344:3: warning: unannotated fall-through between switch labels [-Wimplicit-fallthrough]
                case TK_OP_AND:
                ^
modules/gdscript/gdscript_tokenizer.cpp:344:3: note: insert 'break;' to avoid fall-through
                case TK_OP_AND:
                ^
                break;
```

But it turns out that it's the previous case that was falling through.